ingestion/src/metadata/utils/connections.py

```diff
@@ -754,6 +758,28 @@ def _(connection: ModeClient) -> None:
         )
 
 
+@get_connection.register
+def _(connection: MlflowConnection, verbose: bool = False):
+    from mlflow.tracking import MlflowClient
+
+    return MlflowClientWrapper(
+        MlflowClient(
+            tracking_uri=connection.trackingUri,
+            registry_uri=connection.registryUri,
+        )
+    )
+
+
+@test_connection.register
+def _(connection: MlflowClientWrapper) -> None:
+    try:
+        connection.client.list_registered_models()
```

Comment:
Is this an ml MLFlow specific method that is available on the object bound to connection?